core/src/main/java/bio/terra/pearl/core/service/kit/pepper/LivePepperDSMClient.java

```diff
@@ -69,8 +71,8 @@ public PepperKit fetchKitStatus(UUID kitRequestId) throws PepperApiException, Pe
 
     @Override
     public Collection<PepperKit> fetchKitStatusByStudy(String studyShortcode) throws PepperApiException, PepperParseException {
-        var request = buildAuthedGetRequest("kitstatus/study/%s".formatted(makePepperStudyName(studyShortcode)));
-        var response = retrieveAndDeserializeResponse(request, PepperKitStatusResponse.class);
+        WebClient.RequestHeadersSpec<? extends WebClient.RequestHeadersSpec<?>> request = buildAuthedGetRequest("kitstatus/study/%s" .formatted(makePepperStudyName(studyShortcode)));
```

Comment:
Not sure how I feel about this auto generated type, but I think it's still better than `var`.